src/components/AddItemButton.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function AddItemButton() {
+  const history = useHistory();
+
+  const handleRedirectToAddItem = () => {
+    history.push('/additem');
+  };
+  return (
+    <div className="hidden md:block">
+      <div className="fixed bottom-12 right-1/4">
+        <button
+          type="submit"
+          onClick={handleRedirectToAddItem}
+          className="btn-bg flex items-center border py-3 px-3 hover:bg-green-300 justify-center rounded-md bg-green-600 text-white w-40"
+        >
+          <span className="mr-4">Add an Item</span>
+          <span>
```

Comment:
Why not just using the double arrow icon?

Reply:
Of course. As you can see in our code, there is a new folder called img, you can just drop your svgs there. After that, on the top of the component you can go with import {anyName as ReactLogo} from './styles/iconfilename.svg (for more information check out https://blog.logrocket.com/how-to-use-svgs-in-react/ the part with **Using SVG as a component**)

Reply:
fixed too👍🏻